lib/link.js

```diff
@@ -50,6 +50,14 @@ function link (args, cb) {
   linkPkg(npm.prefix, cb)
 }
 
+function parentFolder (id, folder) {
```

Comment:
The worst case scenario here if a `/` is missing is that it refuses to link the module. And given that it's invalid to start a module name with '@' and not have it be a scoped module, I'm not concerned about that happening.